### ISSUE TITLE
[Feature]: Install and set up Preline UI

### ISSUE THREAD

**Issue by chadstewart** [TechIsHiring/techishiring-website#133]
## Feature Request

**Is your feature request related to a problem? Please describe.**
A clear and concise description of what the problem is. Ex. I have an issue when [...]

Install and set up Preline UI

**Describe the solution you'd like**
A clear and concise description of what you want to happen. Add any considered drawbacks.

To begin the migration from Chakra UI to Preline UI, we need to install and set up Preline UI. This is the issue to accomplish that goal in.

Installation Guide: https://preline.co/docs/frameworks-nextjs.html

**Describe alternatives you've considered**

n/a

**Teachability, Documentation, Adoption, Migration Strategy**
If you can, explain how users will be able to use this and possibly write out a version in the docs.
Maybe a screenshot or design?

n/a

**Comment by dev-phantom** [comment]
Has anyone gotten assigned to this?

**Comment by chadstewart** [comment]
> Has anyone gotten assigned to this?

Nope, you can go ahead with this if you like.

**Comment by chadstewart** [comment]
> Has anyone gotten assigned to this?

Nope. Feel free to assign yourself to this task if you're interested.

**Comment by dev-phantom** [comment]
> > Has anyone gotten assigned to this?
> 
> Nope. Feel free to assign yourself to this task if you're interested.

Okay assign me to this

**Comment by dev-phantom** [comment]
warn  - Attempted to load @next/swc-win32-x64-gnu, but it was not installed
warn  - Attempted to load @next/swc-win32-x64-msvc, but an error occurred: The specified module could not be found.
\\?\C:\Users\devphantom\Documents\techishiring-website\node_modules\@next\swc-win32-x64-msvc\next-swc.win32-x64-msvc.node
error - Failed to load SWC binary for win32/x64, see more info here: https://nextjs.org/docs/messages/failed-loading-swc

please i have been trying to fix this issue since

**Comment by chadstewart** [comment]
> warn - Attempted to load @next/swc-win32-x64-gnu, but it was not installed warn - Attempted to load @next/swc-win32-x64-msvc, but an error occurred: The specified module could not be found. \?\C:\Users\devphantom\Documents\techishiring-website\node_modules@next\swc-win32-x64-msvc\next-swc.win32-x64-msvc.node error - Failed to load SWC binary for win32/x64, see more info here: https://nextjs.org/docs/messages/failed-loading-swc
> 
> please i have been trying to fix this issue since

Sorry it took so long to get back to you @dev-phantom. Do you have the branch you're working on by any chance? I'm honestly not sure what's going on here.

**Comment by chadstewart** [comment]
Just want to make a quick note that instead of Preline, we're going to go with ShadCN instead.

Made a note here on the reasoning: https://github.com/TechIsHiring/techishiring-website/issues/60#issuecomment-1716633683

**Comment by Wayne-Jones** [comment]
@chadstewart Any update on when the PR #164 issues would be corrected and merged in?
